content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -2,9 +2,17 @@
 title: Publishing Experimental Plugin Releases
 layout: developer
 ---
-
-To simplify delivery of beta versions of plugins to interested users, the Jenkins project publishes an _experimental update center_.
-It includes alpha and beta versions of plugins, which aren't usually included in the regular update center.
+The link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin Manager tool] with link:/doc/developer/plugin-development/incrementals/[incremental] builds is often a preferred technique as it benefits the user by:
+- providing additional information about the plugins being downloaded (such as, available updates or security warnings and many more), 
+- enabling easier import into an instance of jenkins by downloading plugins and their respective dependencies into a folder,
+- making it easier to test and explore unreleased plugin versions,
+- providing the user with flexibility to create incremental versions of jenkins core or an upstream plugin with dependencies while also providing the ability to switch between and keep track of incremental versions.
+- allowing the administrator to use incremental builds, without requiring a "beta" release.
+
+The other way, is to simplify the delivery of plugin beta versions of to interested users. 
+The Jenkins project publishes an _experimental update center_.
+The update center includes alpha and beta versions of plugins, which aren't usually included in the regular update center. 
+The following will give detailed explanation of this method:
```

Comment:
```suggestion
NOTE: The experimental update center is deprecated to use in favor of incrementals, as described above. 
```
Let's express preference of incrementals over the experimental update center with a note.